src/protocol/codecs/mod.rs

```diff
@@ -1,3 +1,7 @@
 //! Codec implementations for the Algorand network protocol.
 
+pub mod algomsg;
+pub mod payload;
+mod tagmsg;
+mod topic;
 pub mod websocketcodec;
```

Comment:
Hmmm, we could put here some comment which describes which codec handles which part of the entire message frame.

Reply:
Will be done in a separate commit